features/conversation-list/components/conversation-list/conversation-list.tsx

```
@@ -0,0 +1,117 @@
+function useRefreshHandler(args: { onRefetch?: () => Promise<void> }) {
+  const { onRefetch } = args;
+
+  const isRefetchingRef = useRef(false);
+
+  const handleRefresh = useCallback(async () => {
+    if (isRefetchingRef.current) return;
+    isRefetchingRef.current = true;
+    try {
+      await onRefetch?.();
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the catch clause.


(lint/complexity/noUselessCatch)